### PR TITLE
multi: add testdata for versioned db upgrade.

### PR DESCRIPTION
**Pull request by dnldd**
This adds testdata for the upcoming versioned db upgrade.

### CODE REVIEW

File: client/core/core_test.go [hunk 1]
```diff
@@ -342,6 +342,10 @@ func (tdb *TDB) Backup() error {
 	return nil
 }
 
+func (tdb *TDB) Close() error {
+	return nil
+}
+
```
Comment (JoeGruffins) [inline client/core/core_test.go:348]: This was removed and no longer needed?

**Comment by chappjc** [comment]
@dnldd 454 is in and ready for test data update.  Thanks!

File: client/core/core_test.go (hunk 1 above)
Comment (dnldd) [reply client/core/core_test.go:348]: Yes, removing.

**Comment by dnldd** [comment]
Looks like 454 did not change db structure so the regenerated test data is not different from what already existed since its an empty database. Will push JoeGruffins review fix shortly.

**Comment by chappjc** [comment]
> Looks like 454 did not change db structure so the regenerated test data is not different from what already existed since its an empty database. Will push JoeGruffins review fix shortly.

Then I'm confused about what the test data db is supposed to hold because the encoding of balance changed and introduced a balance set with its own blob encodings too.  With no data in the DB other than the version set, I have a hard time seeing when there would ever by changes to the db file when it contains no data.

**Comment by dnldd** [comment]
The first db upgrade is only ensuring that a db version is set when the upgrade is done, since the version is part of the fundamental structure of the db an empty v0 database as the test data is fine here, this upgrade does not need any additional data in the test data to verify.

**Comment by chappjc** [comment]
OK, gotcha it contains only what changes between versions.

**Pull request merged**